frontend/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "digitales-warten-frontend",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://localhost:5000",
+  "proxy": "http://backend:5000",
```

Comment:
This was a problem for us, as some people wanted to build locally (so they needed to use localhost:5000) and some wanted to use the docker/docker-compose interface for testing (so they needed to use backend:5000). We solved this using environment variables and defaulting to backend:5000. Is something similar possible for you as well?

Reply:
The proxy is not meant to be used in production. Production setup usually uses a reverse proxy in front of both frontend and backend and a "sub-folder" for the api, e.g. `digitalwarten.de/api/v1/<api url>`.

Reply:
ok, I added a new commit that now deploys the API in /api/v1/...

Reply:
Please also revert this change to package.json

Reply:
good catch, done